model_cards/Ogayo/Hel-ach-en/README.md

```diff
@@ -0,0 +1,49 @@
+---
+language: 
+- ach 
+- en
+tags:
+- translation
+- Pytorch
+license: CC By 4.0
```

Comment:
```suggestion
license: cc-by-4.0
```
we use the identifiers from https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/licensing-a-repository#searching-github-by-license-type